packages/amplify_datastore/lib/event_channel_datastore.dart

```diff
@@ -21,12 +21,23 @@ typedef void CancelListening();
 class AmplifyDataStoreEventChannel {
   var events = const EventChannel("com.amazonaws.amplify/datastore_hub_events");
   var stream;
-  listenToDataStore(Listener listener) {
-    stream = events.receiveBroadcastStream(1).listen(listener);
+  listenToDataStore(Listener listener, {Function onError}) {
+    // TODO: Assign as default parameter
+    var errorHandler;
+    if (onError == null) {
+      errorHandler = defaultErrorHandler;
+    } else {
+      errorHandler = onError;
+    }
```

Comment:
Can be replaced with `var errorHandler = onError ?? defaultErrorHandler` or just use `onError ?? defaultErrorHandler` inline.

Reply:
fixed.